committing.rst

```diff
@@ -93,6 +92,8 @@ making a complete patch.
 Commit Style
 ------------
 
+.. move this to pullrequest
```

Comment:
Committing is now done by contributors, so this section doesn't belong here anymore.
Something could be said about the commit message used while squashing.